crates/bevy_render/src/deterministic.rs

```diff
@@ -0,0 +1,14 @@
+use bevy_ecs::system::Resource;
+
+/// Configure deterministic rendering.
+///
+/// Render is parallel by default, and in certain situations it may cause visual artifacts like flickering.
+/// Default fix for the issue is to set `depth_bias` per material.
+/// When it is not possible, deterministic rendering can be used.
+///
+/// This option costs performance and disabled by default.
+#[derive(Resource, Default)]
+pub struct DeterministicRenderingConfig {
+    /// Whether to enable deterministic rendering.
+    pub deterministic: bool,
```

Comment:
I think we should name this something more specific for each fix that we might apply here.

In this case, `stable_sort_z_fighting` or something would be better.